addons/account/models/account_move.py

```diff
@@ -3750,6 +3750,17 @@ def invalidate_cache(self, fnames=None, ids=None):
                 self.env['account.move'].invalidate_cache(ids=move_ids)
         return super().invalidate_cache(fnames=fnames, ids=ids)
 
+    @api.model
+    def _name_search(self, name, args=None, operator='ilike', limit=100, name_get_uid=None):
+        if operator == 'ilike':
```

Comment:
It's a bit sad that we only support one operator, but I guess it's good enough for this "fix"